registry/storage/driver/filesystem/driver.go

```diff
@@ -149,9 +161,15 @@ func (d *driver) PutContent(ctx context.Context, subPath string, contents []byte
 // Reader retrieves an io.ReadCloser for the content stored at "path" with a
 // given byte offset.
 func (d *driver) Reader(ctx context.Context, path string, offset int64) (io.ReadCloser, error) {
+	span, _ := tracing.StartSpan(ctx, fmt.Sprintf("%s:%s", driverName, "Reader"),
+		trace.WithAttributes(attribute.String("path", path),
```

Comment:
we should capitalise "path" or change `GetContent` so we are consistent across all recorded spans/codebase.